core/test/unit/web/parent-app_spec.js

```diff
@@ -0,0 +1,71 @@
+var should = require('should'),
+    sinon = require('sinon'),
+    proxyquire = require('proxyquire'),
+    sandbox = sinon.sandbox.create();
+
+describe('parent app', function () {
+    it('should successfully instantiate the app', function () {
+        should.exist(require('../../../server/web/parent-app')());
+    });
+
+    it('should mount applications', function () {
+        const app = require('../../../server/web/parent-app')();
+        const mountedApps = app._router.stack.filter(r => (r.name === 'mounted_app'));
+        // making sure we register 5 applications:
+        // v0.1, Content API v2, Admin API v2, Admin, Public Site
+        mountedApps.length.should.equal(5);
```

Comment:
Wanted to check what the app does without any mocks/spies etc. Maybe going to far with it?

Reply:
Hm the test below does test basically the same or? We test how often `use` is called with which parameters.
Let me know what you think.

> does without any mocks/spies etc

Hm yeah, that is weird. Same comment as above.